roles/satellite-clone/tasks/restore.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: Restore mongo data
   command: "tar --selinux --overwrite -xf {{ backup_dir }}/mongo_data.tar.gz -C /"
-  when: standard_backup
+  when: rhel_migration or standard_backup
```

Comment:
@sthirugn I think we should mirror katello-restore as much as possible, which works off of the files present rather than the scenario itself. We have checks (which admittedly, as we just discovered, need a little refining) to make sure one backup is present and then correct files are there